### ISSUE TITLE
Intro/startup sound

### ISSUE THREAD

**Issue by AhayriSG** [hrydgard/ppsspp#18393]
### What should happen

Currently PPSSPP doesn't have startup sound.

### Who would this benefit

[PPSSPP's interface sound files](https://github.com/hrydgard/ppsspp/pull/13239) (UISound = true) works pretty well actually. But currently there is no startup sound implementation for PPSSPP.

### Platform (if relevant)

None

### Games this would be useful in

User Interface

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by rockman6554** [comment]
That's sounds great, it would be awesome have these feature

**Comment by AhayriSG** [comment]
> That's sounds great, it would be awesome have these feature

Yeah, also implementing this feature with this way will not infringe copyrighted content unlike mentioned in https://github.com/hrydgard/ppsspp/issues/9214
